internal/setup/setup_test.go

```diff
@@ -888,11 +913,13 @@ var setupTests = []setupTest{{
 					suites: [jammy]
 					v1-public-keys: [extra-key]
 					default: true
+					priority: 20
```

Comment:
Removed the `default: true` because we don't need it.

As per the priority values, if `priority` is unspecified for two or more archive, it will be zero by default and the priorities will conflict. Hence the inclusion here.